addons/account/views/account_move_views.xml

```diff
@@ -628,36 +629,34 @@
-                                <div class="row mb4">
-                                    <div class="col-6 o_kanban_record_headings">
-                                        <strong>
-                                            <span>
-                                                <field name="partner_id" invisible="not partner_id" readonly="state != 'draft'" />
-                                                <field name="journal_id" invisible="partner_id" />
-                                            </span>
-                                        </strong>
-                                    </div>
-                                    <div class="col-6 text-end">
-                                        <strong><i class="fa fa-clock-o" aria-label="Date" role="img" title="Date"/> <t t-out="record.date.value"/></strong>
-                                    </div>
+                        <t t-name="kanban-card">
+                            <div class="row mb4">
+                                <div class="col-6 o_kanban_record_headings">
+                                    <strong>
+                                        <span>
+                                            <field name="partner_id" invisible="not partner_id" readonly="state != 'draft'" />
+                                            <field name="journal_id" invisible="partner_id" />
+                                        </span>
+                                    </strong>
                                 </div>
-                                <div class="row">
-                                    <div class="col-12">
-                                        <span><field name="ref"/></span>
-                                    </div>
+                                <div class="col-6 text-end">
+                                    <strong><i class="fa fa-clock-o" aria-label="Date" role="img" title="Date"/> <field name="date"/></strong>
                                 </div>
-                                <div class="row">
-                                    <div class="col-6">
-                                        <span><field name="amount_total" widget='monetary'/></span>
-                                        <span><field name="currency_id" invisible="1" readonly="state in ['cancel', 'posted']"/></span>
-                                    </div>
-                                    <div class="col-6">
-                                        <span class="float-end">
-                                            <field name="state" widget="label_selection" options="{'classes': {'draft': 'default', 'posted': 'success'}}"/>
-                                        </span>
-                                    </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-12">
+                                    <span><field name="ref"/></span>
```

Comment:
challenge the fact to wrap fields in spans: probably not necessary. Always try to simplify as much as possible